modules/locate.js

```diff
@@ -0,0 +1,130 @@
+                aliases: [],
+                inline: false,
+                description: "Lists stores that are nearby the specified location",
+                help: '',
+                examples: ["!locate New York"]
+            }
+        };
+        log.info("Locate ready");
+    }
+
+    getCommands(){
+        return this.commands;
+    }
+
+    generateRequestBody(geometry){
+        let requestBody = {
+            count: 9,
+            filter_mass_markets: true,
+            language: "en-us",
+            page: 1,
+            request:{
+                EarliestEventStartDate: null,
+                LatestEventStartDate: null,
+                LocalTime: '/Date('+Date.now()+')/',
+                EventTypeCodes: [],
+                PlayFormatCodes: [],
+                MarketingProgramCodes: [],
+                ProductLineCodes: ["MG"],
+                SalesBrandCodes: ["MG"]
+            }
+        };
+        if(geometry.bounds){
+            requestBody.request.North = geometry.bounds.northeast.lat;
+            requestBody.request.East = geometry.bounds.northeast.lng;
+            requestBody.request.South = geometry.bounds.southwest.lat;
+            requestBody.request.West = geometry.bounds.southwest.lng;
+        }else{
+            requestBody.request.North = geometry.location.lat;
+            requestBody.request.East = geometry.location.lng;
+            requestBody.request.South = geometry.location.lat;
+            requestBody.request.West = geometry.location.lng;
+        }
+        return requestBody;
+    }
+
+    generateEmbed(results,googleResult){
+        const descriptions = [];
+        const googleStaticMap = [this.googleStaticMap];
+        results.forEach(result=>{
+            const description = {name:["*",(results.indexOf(result)+1),": ",result.Address.Name,"*","\n"].join(""),
+                inline:true};
+            const descriptionValue = [result.Address.Line1," - ",result.Address.City,"\n"];
+            const url = [this.wizardsStoreUrl];
+            url.push("p=",result.Address.City,",+",result.Address.CountryName);
+            url.push("&c=",result.Address.Latitude,",",result.Address.Longitude);
+            url.push("&loc=",result.Id);
+            url.push("&orgid=",result.Organization.Id);
+            url.push("&addrid=",result.Address.Id);
+            descriptionValue.push("[Event Page]","(",encodeURI(url.join("")),")");
+            description.value=descriptionValue.join("");
+            descriptions.push(description);
+
+            googleStaticMap.push(this.googleStaticMapStandardMarkup);
+            googleStaticMap.push("label:",(results.indexOf(result)+1),"|");
+            googleStaticMap.push(result.Address.Line1," ",result.Address.PostalCode," ",result.Address.City," ",
+                result.Address.CountryName);
+        });
+        return rp({url: encodeURI(googleStaticMap.join("")),encoding:null}).then(body=>{
+            return new Discord.RichEmbed({
```

Comment:
If the only thing you're doing in an anonymous inline function is returning a variable, you can omit the curly brackets:
`.then(body => new Discord.RichEmbed({...}));`